docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -36,6 +36,14 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
     [ LOCATION path ]
```

Comment:
The bucketSpec is still missing in CREATE HIVE FORMAT table, right?
```
    [ CLUSTERED BY ( col_name3, col_name4, ... ) 
        [ SORTED BY ( col_name [ ASC | DESC ], ... ) ] 
        INTO num_buckets BUCKETS ]
```

Reply:
Any reason we did not add it? @huaxingao @GuoPhilipse

Reply:
Sorry, we missed that. 
@GuoPhilipse Could you please have a follow-up to add ```bucketSpec```? Thanks!

Reply:
Thank you!

Reply:
Yes, we need it, will add it soon.